### PR TITLE
Delegate crashes

### PR DESCRIPTION
**Pull request by antrix1989**

### CODE REVIEW

File: Pod/Classes/ViewController.swift [hunk 1]
```diff
@@ -252,8 +252,8 @@ public class ViewController:UIViewController, UIScrollViewDelegate {
         let screenWidth = UIScreen.mainScreen().bounds.size.width
         var movedHeight = CGFloat(0)
         
-        if self.delegate!.respondsToSelector("photoSliderControllerWillDismiss:") {
-            self.delegate!.photoSliderControllerWillDismiss!(self)
+        if (self.delegate?.respondsToSelector("photoSliderControllerWillDismiss:") != nil) {
```
Comment (nakajijapan) [inline Pod/Classes/ViewController.swift:255]: @antrix1989  I think `(` is unnecessary in Swift.

**Comment by seapy** [comment]
When this feature is merged or fixed?
some situation delegate is not required so `delegate!` to be `delegate?`

File: Pod/Classes/ViewController.swift (hunk 1 above)
Comment (nakajijapan) [inline Pod/Classes/ViewController.swift:255]: wow, xcode automatically added the parenthesis!!!!!

**Comment by nakajijapan** [comment]
OK, merged.

**Pull request merged**